templates/search/blocks/search_sort_and_view_options.html

```diff
@@ -5,16 +5,16 @@
 <!--    <p class="search-results__explainer">Results for everything on our website that matches your search term.</p>-->
     <form method="GET" class="search-sort-view__form search-sort-view__desktop" data-id="sort-form-desktop" id="catalogue-sort-form-desktop" data-search-type="{{ view.search_tab }}" data-search-bucket="{{ buckets.current.label }}" data-search-filter-name="Sort results" data-search-filter-value="">
 
-        <h2 class="tna-heading-l search-sort-view__heading">
-           <label for="id_sort_by_desktop">Sort by</label>
+        <h2 class="tna-heading-l search-sort-view__heading sr-only">
+           <label for="id_sort_by_desktop">Sort byx</label>
```

Comment:
```suggestion
           <label for="id_sort_by_desktop">Sort by</label>
```

Reply:
Thanks @TNA-Allan, good catch I've now fixed that and tests are passing. If you could review and approve please?